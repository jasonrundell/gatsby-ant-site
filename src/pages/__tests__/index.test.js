import React from 'react'
import renderer from 'react-test-renderer'

import Index from '../index'

describe('Index', () => {
  it('renders correctly', () => {
    const data = {
      site: {
        siteMetadata: {
          title: 'Default Starter Blog',
        },
      },
      file: {
        childImageSharp: {
          fluid: {
            base64: '',
            aspectRatio: 1,
            src: '',
            srcSet: '',
            sizes: '',
          },
        },
      },
    }

    const tree = renderer.create(<Index data={data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
