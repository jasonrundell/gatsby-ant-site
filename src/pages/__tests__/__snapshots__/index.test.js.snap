// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div
      className="container"
    >
      <h1
        className="title"
      >
        <a
          className="link"
          href="/"
        >
          Default Starter Blog
        </a>
      </h1>
    </div>
  </header>,
  <div
    className="container"
  >
    <main>
      <h1>
        Hi people
      </h1>
      <p>
        Welcome to your new Gatsby site.
      </p>
      <p>
        Now go build something great.
      </p>
      <div
        className="containerFigure"
      >
        <figure>
          <div
            className=" gatsby-image-wrapper"
            style={
              Object {
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              style={
                Object {
                  "paddingBottom": "100%",
                  "width": "100%",
                }
              }
            />
            
            <picture>
              <source
                sizes=""
                srcSet=""
              />
              <img
                alt=""
                loading="lazy"
                onLoad={[Function]}
                sizes=""
                src=""
                srcSet=""
                style={
                  Object {
                    "height": "100%",
                    "left": 0,
                    "objectFit": "cover",
                    "objectPosition": "center",
                    "opacity": 0,
                    "position": "absolute",
                    "top": 0,
                    "transition": "opacity 500ms",
                    "width": "100%",
                  }
                }
              />
            </picture>
            <noscript
              dangerouslySetInnerHTML={
                Object {
                  "__html": "<picture><source srcset=\\"\\" /><img loading=\\"lazy\\" src=\\"\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture>",
                }
              }
            />
          </div>
          
        </figure>
      </div>
      <a
        href="/page-2/"
      >
        Go to page 2
      </a>
    </main>
    <footer>
      Â© 
      2019
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
        rel="noopener noreferrer"
      >
        Gatsby
      </a>
    </footer>
  </div>,
]
`;
